### ISSUE TITLE
bug: Watch in VR button (for Google Cardboard) doesn't show even after disabling "ReVance/Player/Flyout menu/Hide Watch in VR" setting

### ISSUE THREAD

**Issue by takeda** [ReVanced/revanced-patches#3640]
### Bug description

I'm trying to watch YouTube 360 video with Google cardboard, things seem to work fine in regular YouTube, but in ReVanced the VR icon is missing. I eventually found option to enable it under "ReVance/Player/Flyout menu/Hide Watch in VR" but it doesn't seem to do anything.

### Additional context

I'm using ReVanced 4.13.3 with youtube 19.16.39, with all default patches selected.

### Acknowledgements

- [X] I have checked all open and closed bug reports and this is not a duplicate.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Exclude patches until the error is gone, report back which one caused it. Start with the spoof client patch

**Comment by takeda** [comment]
BTW: these are unrelated but I also observed 2 bugs:

- sometimes when I finished patching and clicked install, it was telling me "Installation cancelled" and it was happening every time I click on the button, I had to close ReVanced manager, and re-patch again? Is there a way to install the generated apk without repatching it (i.e. where it is located?)
- another issue was that it looks like some kind of memory leak, it doesn't affect most people because they only patch it and exit the app, but I started getting crashes during patching (the app continued to run but displayed traceback) because ReVanced Manager run out of memory. I had to terminate process every time I started patching to prevent this from happening.

**Comment by oSumAtrIX** [comment]
Send a recording of how you patch yt with gmscore only and it doesn't appear but does on unpatched app. Same version

**Comment by takeda** [comment]
How can I record my screen on android? I never did that.

Anyway I was continuing experimenting and restored all patches back. In Misc menu under GmsCore Settings I saw spoof client and inside of it that my phone is pretending to be iOS. When researching this issue I found people complaining that Google removed VR from iOS version of YouTube, so I decided to uncheck it. I guess that's why you were saying to remove that patch first. And the cardboard icon showed up. It overlays the one to copy playback position though so I can't click it, but I found that I still might be able to select it in the menu.

Now I understand, you told me to remove patches and start with client spoof one, but that's exactly what I did and I know I was removing them because I saw annoying "features" coming back.

I had one more theory so I did some additional test to confirm it. I first tried left it in iOS mode, then removed all patches (except gmscore) and the feature wasn't available, then I brought all patches back, disabled "iOS spoofing" and once again I removed all patches (except gmscore) and this time the cardboard icon was working. So I believe the setting is retained even if patch is not present.

So I guess my issue was caused by the iOS spoofing as you originally suspected (why ReVanced is doing it by default?), and another one is the overlap (although that might be less of a problem, since I can also use the cog wheel to select VR there). I can disable the timestamp copy, although that feature is also useful. Is it possible to move the copy to the left, or it's that not possible when app is patched?
